interfaces/Portalicious/staticwebapp.config.json

```diff
@@ -2,39 +2,25 @@
   "$schema": "https://json.schemastore.org/staticwebapp.config.json",
   "routes": [
     {
-      "route": "/*.js",
+      "route": "*.{js,json,css,jpg,png,ico,webmanifest,tff,woff,woff2}",
       "headers": {
         "Cache-Control": "immutable, no-transform, max-age=31536000"
       }
     },
     {
-      "route": "/*.css",
-      "headers": {
-        "Cache-Control": "immutable, no-transform, max-age=31536000"
-      }
+      "route": "/index.html",
```

Comment:
I think by using: `"route": "/*"`, we could 'auto upgrade/redirect' all existing URLs, hopefully?

Reply:
I think we can test this locally by using: https://github.com/Azure/static-web-apps-cli

I wonder if its worth it to use that as our "local dev server" and/or "production-mode test-tool"... As it will also check/use/fake the configurations around headers, cache-settings, etc.. I think.

Reply:
Haha, I didn't see your other PR yet about the SWA-CLI :)

But maybe, to prevent yet another fix-rebase-dance; lets combine #5585 into this branch and to a final rebase. ;)